torch/utils/benchmark/examples/sparse/op_benchmark.py

```diff
@@ -0,0 +1,98 @@
+"""Example use of Timer and sparse op fuzzers to measure kernel performance.
+
+$ python -m examples.sparse.op_benchmark
+"""
+
+import numpy as np
+import torch
+
+from torch.utils.benchmark import Timer
+from torch.utils.benchmark.op_fuzzers.sparse_unary import UnaryOpSparseFuzzer
+from torch.utils.benchmark.op_fuzzers.sparse_binary import BinaryOpSparseFuzzer
+
+_MEASURE_TIME = 1.0
+
+def assert_dicts_equal(dict_0, dict_1):
+    """Builtin dict comparison will not compare numpy arrays.
+    e.g.
+        x = {"a": np.ones((2, 1))}
+        x == x  # Raises ValueError
+    """
+    assert set(dict_0.keys()) == set(dict_0.keys())
+    assert all(np.all(v == dict_1[k]) for k, v in dict_0.items() if k != "dtype")
+
+def run(n, stmt, fuzzer_cls):
+    float_iter = fuzzer_cls(seed=0, dtype=torch.float32).take(n)
+    int_iter = fuzzer_cls(seed=0, dtype=torch.float64).take(n)
```

Comment:
this does not generate integer tensors, so don't call it `int_iter`

Reply:
ok